frontend/src/components/visual-editor/form/inputs/message/QuickRepliesInput.tsx

```diff
@@ -47,17 +47,7 @@ const QuickRepliesInput: FC<QuickRepliesInput> = ({
     const updatedQuickReplies = [...quickReplies];
 
     updatedQuickReplies.splice(index, 1);
-    setQuickReplies(
-      updatedQuickReplies.length
-        ? updatedQuickReplies
-        : [
-            createValueWithId({
-              content_type: QuickReplyType.text,
-              title: "",
-              payload: "",
-            }),
-          ],
-    );
+    setQuickReplies(updatedQuickReplies.length ? updatedQuickReplies : []);
```

Comment:
Thank you for your contribution 🥳 

How about we add a condition at the useEffect() level. If there's only 1 quick reply and is empty, then we trigger onChange with a empty array ?

Reply:
thanks for the suggestion will do it

Reply:
@marrouchi done the changes requested by you please review